docs/source/topics/proc_setting-up-remote-server.adoc

```diff
@@ -69,8 +69,8 @@ defaults
     log global
     mode http
     timeout connect 5000
-    timeout client 5000
-    timeout server 5000
+    timeout client 500000
+    timeout server 500000
```

Comment:
Shouldn't we just document it instead making it default `5000 => 500000` ?

Reply:
this is about 8 and a half minutes to get a timeout.
Wouldn't in this case the client not just hang for a long time until it gets an answer... not sure what would be the better user experience.

Reply:
The documentation could be updated to represent this, but as a user I would expect the documentation to provide an experience where I can perform "normal" operations like tailing logs without receiving an EOF.